bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3070,7 +3074,7 @@ void ReplicaImp::goToNextView() {
 bool ReplicaImp::tryToEnterView() {
   ConcordAssert(!currentViewIsActive());
   ConcordAssertEQ(activeExecutions_, 0);
-  std::vector<PrePrepareMsg *> prePreparesForNewView;
+  std::vector<std::shared_ptr<PrePrepareMsg>> prePreparesForNewView;
```

Comment:
Why is it shred? I think you might be able to make it unique, that's the default pointer type.